components/Form.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function Form() {
+  const [title, setTitle] = useState('');
+  const [lyrics, setLyrics] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    setTitle(event.target.title.value.trim());
+    setLyrics(event.target.lyrics.value.trim());
+    form.reset();
+    form.title.focus();
+  }
+
+  return (
+    <>
+      <InputForm onSubmit={handleSubmit}>
+        Start creating your first idea:
+        <div>
+          <label for="title">Title: </label>
+          <input type="text" id="title" name="title" required />
+        </div>
+        <div>
+          <LyricsField
+            type="text"
+            placeholder="Keep your ideas rolling!"
+            id="lyrics"
+            name="lyrics"
+            required
+          />
+        </div>
+        <Button type="submit">Done!</Button>
+      </InputForm>
+      <OutputContainer>
+        <Output1>{title ? title : ''}</Output1>
+        <Output2>{lyrics ? lyrics : ''}</Output2>
+      </OutputContainer>
```

Comment:
```suggestion
      {(title && lyrics) && 
      <OutputContainer>
        <Output1>{title}</Output1>
        <Output2>{lyrics}</Output2>
      </OutputContainer>
      }
```
With this change you render the Outputcontainer only in case you have an output. Also you don`t need to check if there is a title or  lyrics in Output1 and Output2 because thats its already checked that :)